### PR TITLE
Add: step_relevel

### PR DESCRIPTION
**Pull request by dimagor**
Adding `step_relevel()` - #438

### CODE REVIEW

File: R/relevel.R [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+#'   prep()
+#'
+#' data <- bake(rec, okc)
+#' levels(data$diet)
+step_relevel <-
+  function(recipe,
+           ...,
+           role = NA,
+           trained = FALSE,
+           ref_level = "UNKNOWN",
```
Comment (topepo) [inline R/relevel.R]: I would avoid having a default value for `ref_level`; that will force people to specify one.

File: R/relevel.R [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+      id = id
+    )
+  }
+
+#' @export
+prep.step_relevel <- function(x, training, info = NULL, ...) {
+  col_names <- recipes::terms_select(x$terms, info = info)
+  col_check <- dplyr::filter(info, .data$variable %in% col_names)
+  if (any(col_check$type != "nominal")) {
+    stop(
```
Comment (topepo) [inline R/relevel.R]: Please use `rlang::abort()` instead of `stop()`

**Comment by topepo** [comment]
Looks good. I added a few minor comments. I'll be sending to CRAN on 1/6.

**Comment by dimagor** [comment]
@topepo Thanks for the review. Requested changes have been made & pushed.

**Comment by topepo** [comment]
Thanks!

**Pull request merged**